specs/deneb/p2p-interface.md

```diff
@@ -119,7 +120,7 @@ The *type* of the payload of this topic changes to the (modified) `SignedBeaconB
 
 ###### `blob_sidecar_{index}`
 
-This topic is used to propagate signed blob sidecars, one for each sidecar index. The number of indices is defined by `MAX_BLOBS_PER_BLOCK`.
+This topic is used to propagate signed blob sidecars, one for each sidecar index. The number of indices is defined by `MAX_BLOBS_PER_BLOCK` which can be upgraded on via a fork. However the actual blobs that might be generated by the EL will be limited by `MAX_DATA_GAS_PER_BLOCK // DATA_GAS_PER_BLOB` (requires an EL hardfork to upgrade the data gas limit) but should never exceed `MAX_BLOBS_PER_BLOCK` limit of the corresponding CL fork.
```

Comment:
I don't think mentioning the EL constants in the consensus spec is a good idea, since I think we should make the EL and CL as independent to each other as possible.

Reply:
don't have an opinion on this, however someone asked for a clarification note to be included as how all these constants come together, may be this can be move to beacon chain than p2p interface

Reply:
I can find the connection between these cross layer variables useful to be outlined. It would facilitate easier understanding of the link between them when they are about to be changed. Maybe we can clarify this in the EIP since EIPs becomes a place where rationales and design notes for both layers are located.

Reply:
> Maybe we can clarify this in the EIP since EIPs becomes a place where rationales and design notes for both layers are located.

I agree to put it in the EIP instead.